### PR TITLE
PR for ex-2

### PR DESCRIPTION
**Pull request by FinalAngel**

### CODE REVIEW

File: ex2-solution/src/components/app/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Container, Header, Footer, Copyright } from "../../components/layout";
```
Comment (vxsx) [inline ex2-solution/src/components/app/index.js:3]: i would prefer `../layout` or better `components/layout` which needs the modules root set

File: ex2-solution/src/components/layout/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <Logo />
+    <Navigation />
+  </header>
+);
+
+export const Footer = () => (
+  <footer className="footer">
+    <h2 hidden>Footer</h2>
+    <QuickLinks
+      items={["Home", "Events", "Support", "Contact"]}
```
Comment (vxsx) [inline ex2-solution/src/components/layout/index.js:31]: no urls :trollface:

File: ex2-solution/src/components/navigation/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import Nav from "./nav";
+import NavItem from "./nav-item";
+
+import "./navigation.scss";
+
+const items = ["Home", "Event", "Accessoires", "Apparel", "Feautred Items"];
```
Comment (vxsx) [inline ex2-solution/src/components/navigation/index.js:8]: featured typo

File: ex2/src/components/common/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components";
+
+// example using styled-components
+const StyledDiv = styled.div`
+  width: 800px;
+  margin: 20px auto;
+`;
+
+export const Container = ({ children }) => <StyledDiv>{children}</StyledDiv>;
```
Comment (vxsx) [inline ex2/src/components/common/index.js:10]: it's sorta pointless

same as 

```
export const Container = styled.div`
    width: 800px;
    margin: 20px auto;
`;
```

File: ex2/src/styles.scss [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+h3 {
+  margin: 0 0 10px;
+}
+
+.header {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+
+  .meta {
```
Comment (vxsx) [inline ex2/src/styles.scss:14]: why nest

Comment (vxsx) [reply ex2/src/styles.scss:14]: ah, this is original, nevermind

File: ex2-solution/src/components/layout/index.js (hunk 2 above)
Comment (FinalAngel) [reply ex2-solution/src/components/layout/index.js:31]: meh, sample 👍

**Pull request closed without merging**